benchmark/src/main/java/org/corfudb/benchmark/Common.java

```diff
@@ -0,0 +1,27 @@
+package org.corfudb.benchmark;
+
+import org.apache.commons.math3.random.BitsStreamGenerator;
+import org.apache.commons.math3.random.MersenneTwister;
+import org.corfudb.runtime.CorfuRuntime;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.State;
+
+import static org.openjdk.jmh.annotations.Scope.Benchmark;
+
+@State(Benchmark)
+public class Common {
+    final BitsStreamGenerator RND = new MersenneTwister();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "RND" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)